test/unit/query-interface/decrement.test.ts

```diff
@@ -0,0 +1,48 @@
+import { DataTypes } from '@sequelize/core';
+import { expect } from 'chai';
+import sinon from 'sinon';
+import { expectsql, sequelize } from '../../support';
+
+describe('QueryInterface#decrement', () => {
+  const User = sequelize.define('User', {
+    firstName: DataTypes.STRING,
+  }, { timestamps: false });
+
+  afterEach(() => {
+    sinon.restore();
+  });
+
+  // you'll find more replacement tests in query-generator tests
+  it('does not parse replacements outside of raw sql', async () => {
+    const stub = sinon.stub(sequelize, 'queryRaw');
+
+    await sequelize.getQueryInterface().decrement(
+      User,
+      User.tableName,
+      // where
+      { id: ':id' },
+      // incrementAmountsByField
+      { age: ':age' },
+      // extraAttributesToBeUpdated
+      { name: ':name' },
+      // options
+      {
+        returning: [':data'],
+        replacements: {
+          age: 1,
+          id: 2,
+          data: 3,
+        },
+      },
+    );
+
+    expect(stub.callCount).to.eq(1);
+    const firstCall = stub.getCall(0);
+    expectsql(firstCall.args[0] as string, {
+      postgres: `UPDATE "Users" SET "age"="age"- ':age',"name"=':name' WHERE "id" = ':id' RETURNING ":data"`,
```

Comment:
Not sure how I can get a comment on multiple lines, but can you swap `postgres` here with `default`?

Reply:
click on one line, then shift+click on another one to comment on multiple lines :)

Reply:
click one line, then shift+click another to comment on multiple lines :)